web/src/components/storage/DASDPage.tsx

```diff
@@ -43,7 +43,7 @@ export default function DASDPage() {
       </Page.Content>
 
       <Page.Actions>
-        <Page.Back>{_("Close")}</Page.Back>
+        <Page.Cancel navigateTo={PATHS.targetDevice}>{_("Back")}</Page.Cancel>
```

Comment:
Do you really navigate to a specific path or it's enough going Page.Bavk?

Just a question